index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rock-Papper-Scissors</title>
+
+</head>
+<body>
+    <h1>Rock-Papper-Scissors Game</h1>
+    <p>Game played 5 times</p>
+   <div id="player"></div>
+   <div id="computer"></div>
+   <div id="player-score"></div>
+   <script src = "index.js"></script>
```

Comment:
pls be mindful of the formatting - repeating again - use [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) with VsCode